### PR TITLE
Add release dependencies

### LINKED ISSUE #60: Add releases dependency graph
Allow release to depend on another release

### PR DESCRIPTION
**Pull request by r3nic1e**
Add support for releases dependencies without checking for cyclic dependencies

Resolves #60

### CODE REVIEW

File: pkg/yml/config.go [hunk 1]
```diff
@@ -6,8 +6,9 @@ import (
 )
 
 type Config struct {
-	Project      string
-	Version      string
-	Repositories []*repo.Config
-	Releases     []*release.Config
+	Project            string
+	Version            string
+	EnableDependencies bool
```
Comment (zhilyaev) [inline pkg/yml/config.go:11]: `EnableDependencies` will be unmarshal to helmwave.yml.tpl

File: pkg/helmwave/cli.go [hunk 2]
```diff
@@ -63,6 +63,8 @@ func (c *Config) plan(opts *yml.SavePlanOptions) error {
 
 	opts.Tags(c.Tags.Value())
 
+	c.Yml.EnableDependencies = c.EnableDependencies
```
Comment (zhilyaev) [inline pkg/helmwave/cli.go:66]: YML.CONFIG

File: pkg/helmwave/config.go [hunk 3]
```diff
@@ -9,13 +9,14 @@ import (
...
-	Kubedog  *kubedog.Config
+	Version            string
+	Helm               *helm.EnvSettings
+	Tags               cli.StringSlice
+	Tpl                template.Tpl
+	Yml                yml.Config
+	PlanPath           string
+	Logger             *Log
+	Parallel           bool
+	EnableDependencies bool
```
Comment (zhilyaev) [inline pkg/helmwave/config.go:20]: EnableDependencies -> yml.EnableDependencies?

**Comment by r3nic1e** [comment]
@zhilyaev fixed

**Pull request merged**